alembic/versions/2024_05_22_2011-2ea9f6f0b790_make_wcs_a_fileondiskmixin.py

```diff
@@ -0,0 +1,46 @@
+"""make WCS a FileOnDiskMixin
+
+Revision ID: 2ea9f6f0b790
+Revises: ec64a8fd8cf3
+Create Date: 2024-05-22 20:11:31.195961
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '2ea9f6f0b790'
+down_revision = '9a4097979249'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('cutouts_q3c_ang2ipix_idx', table_name='cutouts')
+    op.drop_index('exposure_q3c_ang2ipix_idx', table_name='exposures')
+    op.drop_index('images_q3c_ang2ipix_idx', table_name='images')
+    op.drop_index('measurements_q3c_ang2ipix_idx', table_name='measurements')
```

Comment:
Alembic doesn't know about q3c indices so it always tries to remove them on each revision. Make sure you get rid of these (in the downgrade, too).